<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ứng Dụng Notes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QFrame" name="navigationBar">
      <property name="styleSheet">
       <string>
        QFrame {
            background-color: #F9F5E7;
            border: 1px solid #E2DFD2;
            border-radius: 10px;
        }
        QPushButton {
            background-color: transparent;
            border: none;
            font-size: 14px;
            font-weight: bold;
            color: #333333;
        }
        QPushButton:hover {
            color: #FF9800;
        }
       </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QHBoxLayout" name="navLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="notesButton">
         <property name="text">
          <string>Ghi Chú</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loginButton">
         <property name="text">
          <string>Đăng Nhập</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="signupButton">
         <property name="text">
          <string>Đăng Ký</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="notesPage">
       <layout class="QVBoxLayout" name="notesLayout">
        <item>
         <widget class="QTextEdit" name="noteEditor">
          <property name="styleSheet">
           <string notr="true">
            QTextEdit {
                border: 1px solid #E2DFD2;
                border-radius: 10px;
                padding: 15px;
                font-size: 16px;
                background-color: #FFFFFF;
color: black;
            }
           </string>
          </property>
          <property name="placeholderText">
           <string>Nhập ghi chú của bạn...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QPushButton" name="saveButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="loadButton">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="loginPage">
       <layout class="QVBoxLayout" name="loginLayout">
        <item>
         <widget class="QLabel" name="loginTitle">
          <property name="styleSheet">
           <string>
            QLabel {
                font-size: 20px;
                font-weight: bold;
                color: #333333;
            }
           </string>
          </property>
          <property name="text">
           <string>Đăng Nhập</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameInput">
          <property name="placeholderText">
           <string>Tên đăng nhập</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordInput">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Mật khẩu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submitLoginButton">
          <property name="text">
           <string>Đăng Nhập</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="signupPage">
       <layout class="QVBoxLayout" name="signupLayout">
        <item>
         <widget class="QLabel" name="signupTitle">
          <property name="styleSheet">
           <string>
            QLabel {
                font-size: 20px;
                font-weight: bold;
                color: #333333;
            }
           </string>
          </property>
          <property name="text">
           <string>Đăng Ký</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="signupUsernameInput">
          <property name="placeholderText">
           <string>Tên đăng nhập</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="signupPasswordInput">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Mật khẩu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submitSignupButton">
          <property name="text">
           <string>Đăng Ký</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
